R/clean_names.R

```diff
@@ -81,14 +82,19 @@ clean_names.default <- function(dat, ...) {
   if (is.null(names(dat))) {
     dimnames(dat) <- lapply(dimnames(dat), make_clean_names, ...)
   } else {
+    if (set_labels){
+      old_names <- names(dat)
+      for (i in seq_along(old_names)) attr(dat[[i]], "label") <- old_names[[i]]
```

Comment:
Please reformat the code so that the `attr()<-` line is a separate line with braces (`{}`) around it.

Reply:
Done